src/components/buttons/Button.stories.ts

```diff
@@ -27,7 +31,7 @@ const meta: Meta<typeof Button> = {
   },
   parameters: {
     docs: {
-      controls: { exclude: ['prefix', 'suffix', 'click'] },
+      controls: { exclude: ['prefix-icon', 'suffix-icon', 'default'] },
```

Comment:
I think we don't need the icon part in the name, prefix and suffix can take anything not only icons, even though we will use them for icons most of the time.

Reply:
so, prefix is a read-only prop in html buttons, and it seems Storybook passes it to the Vue component as a prop.
and there's a warning on the console about it.
it took me quite some time to figure out

Reply:
I'll probably use something like `preffix` because the Vue warning won't let me use `prefix` since it is a read-only prop in html

Reply:
or, I'll use `prepend` and `append`